dotcom-rendering/stories/generated/CardReadme.stories.jsx

```diff
@@ -0,0 +1,28 @@
+
+import { css } from '@emotion/react';
+
+const ReadMe = () => (
+	<section css={css`
+		padding: 1rem;
+		& h1, p{
+			margin-bottom: 0.5rem;
+		}
+		`}>
+		<h1 css={css`font-size: 1.5em; font-weight: 400;`}>Readme</h1>
+		<p>
+			The stories in this directory are automatically generated.
+		</p>
+		<p>
+			To add new format variations to test, please edit the list in `get-stories.js`.
+		</p>
+	</section>
+);
+
+// eslint-disable-next-line import/no-default-export
+export default {
+	title: 'Components/Card/Format Variations',
```

Comment:
I’m not sure this is the right location for it… [it seems to have only a single variation currently](https://637e40d1bc73bf3f604394b9-wboekubbsr.chromatic.com/?path=/story/components-card-format-variations--readme)?

Reply:
Yep that was introduced by [this PR](https://github.com/guardian/dotcom-rendering/pull/6761) earlier today, afaik 👍 

I believe the current intention is that we will expand the list of variations again, but on a per-case basis rather than generating all possible combinations as we were before. That said, from the discussion in that PR it sounds like we're uncertain about how we intend to use the format variation stories in the future.